Sources/SwiftFCSDKiOS/SwiftFCSDKiOS.docc/Version-4.2.0.md

```diff
@@ -0,0 +1,83 @@
+# Version 4.2.0
+
+This article describes changes in version 4.2.0 of FCSDKiOS
+
+## Overview
+Version 4.2.0 reaches a new milestone in terms of Video Calls. We now support Picture in Picture and Virtual Background. The following Topics are the changes that we have made.
+
+### Preview Buffer View
+This method is designed to create a metal rendered `UIView` for your local video during a video call. This method must be used if you desire to use our **Virtual Background** feature.
+When a call is finished the **SDK** consumer is required to clean up the call by calling the `removePreviewView` method. 
+```swift
+@available(iOS 15, *)
+@MainActor @objc final public func previewBufferView() async -> UIView?
+```
+
+```swift
+@available(iOS 15, *)
+@objc final public func removePreviewView() async
+```
+
+### Sample Buffer View
+This method is designed to create a metal rendered `UIView` for your remote video during a video call. This method must be used if you desire to use our **Picture in Picture** feature.
+When a call is finished the **SDK** consumer is required to clean up the call by calling the `removeBufferView` method. 
+```swift
+@available(iOS 15, *)
+@MainActor @objc final public func sampleBufferView() async -> UIView?
+```
+
+```swift
+@available(iOS 15, *)
+@objc final public func removeSampleView() async
+```
+
+### AVCaptureSession
+In order to check if your devices support Multitasking Camera Access, when a **PreviewBufferView** is created we can get the **AVCaptureSession** from the **PreviewBufferView**. If your Device is supported by Apple, you can use the new **AVPictureInPictureVideoCallViewController** provided by Apple. Otherwise you will need to continue using **AVPictureInPictureController** and request permission from Apple for the proper entitlement. This method only works if you are using **PreviewBufferView** for your Local Video UIView.
```

Comment:
See comment above - same applies here re meaning - "If your Device is supported by Apple" is OK but replace Device with device.